securedrop-keyring/debian/preinst

```diff
@@ -4,10 +4,11 @@ set -e
 
 # Solution adapted from DKG's work on `deb.torproject.org-keyring` and
 # the securedrop core keyring package.
-# In SecureDrop Workstation versions before 0.3.0, the salt provisioning
```

Comment:
On a prod-installed system, I do see a redundant copy of the key in `/etc/apt/trusted.gpg`, so contrary to the old code comment, I believe this issue is present on recent installs due to the `fpf-apt-repo` Salt state, which uses `pkgrepo.managed` and therefore can be expected to write to `/etc/apt/trusted.gpg`:
https://github.com/freedomofpress/securedrop-workstation/blob/main/dom0/fpf-apt-repo.sls#L41-L50